experiments/ssl4eo/sample_conus.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import argparse
+import csv
+import os
+import random
+
+import fiona
+from shapely.geometry import Point, shape
+from torchvision.datasets.utils import download_and_extract_archive
+
+
+def get_uniform_points_within_conus(
+    download_root: str, num_samples: int
+) -> list[tuple[float, ...]]:
+    random.seed(0)
+    nation_url = (
+        "https://www2.census.gov/geo/tiger/GENZ2022/shp/cb_2022_us_nation_5m.zip"
+    )
+    nation_filename = "cb_2022_us_nation_5m.shp"
+    if not os.path.exists(os.path.join(download_root, os.path.basename(nation_url))):
```

Comment:
No need for these if-statements, torchvision already checks to see if the file has already been downloaded and exits immediately if so: https://github.com/pytorch/vision/blob/v0.15.1/torchvision/datasets/utils.py#L127